src/model/Space.cpp

```diff
@@ -129,6 +129,7 @@
 #include <algorithm>
 #include <cmath>
 #include <iterator>
+#include <ranges>
```

Comment:
I've been wary of bringing ranges into OpenStudio due to MSVC lagging behind in supporting them, but I guess it works now! yay!